spec.html

```diff
@@ -40,6 +40,10 @@ <h2 class="nojs">Specification Version 1.0</h2>
                     <td class="t">double</td>
                     <td>8 bytes (64-bit IEEE 754-2008 binary floating point)</td>
                   </tr>
+                  <tr>
+                    <td class="t">decimal128</td>
+                    <td>16 bytes (128-bit IEEE 754-2008 decimal floating point)</td>
```

Comment:
I'd add say: (128-bit IEEE 754-2008 decimal floating point in binary integer decimal (BID) format